ansible_collections/arista/avd/roles/eos_designs/templates/mlag/router-bgp.j2

```diff
@@ -7,6 +7,7 @@ router_bgp:
       type: ipv4
       remote_as: "{{ switch.bgp_as }}"
       next_hop_self: true
+      description: {{ switch.mlag_peer | arista.avd.default(switch.mlag_peer_l3_ip, switch.mlag_peer_ip) }}
```

Comment:
no need to use default filter here. `switch.mlag_peer` must be set at this stage, otherwise it will fale below (line 35)

Reply:
Fixed